test/parser_test.exs

```diff
@@ -0,0 +1,554 @@
+defmodule ParserTest do
+  use ExUnit.Case
+
+  alias Thrift.Parser.Models.Constant
+  alias Thrift.Parser.Models.Exception
+  alias Thrift.Parser.Models.Field
+  alias Thrift.Parser.Models.Function
+  alias Thrift.Parser.Models.Include
+  alias Thrift.Parser.Models.Namespace
+  alias Thrift.Parser.Models.Service
+  alias Thrift.Parser.Models.Struct
+  alias Thrift.Parser.Models.StructRef
+  alias Thrift.Parser.Models.TEnum
+  alias Thrift.Parser.Models.Union
+
+  import ExUnit.CaptureIO
+
+  defp parse(doc, path) do
```

Comment:
Should this function (and the one below) be part of the library's public API (in `lib/parser.ex`)?